src/umbraco.sln

```diff
@@ -123,15 +123,13 @@ Global
 		{31785BC3-256C-4613-B2F5-A1B0BDDED8C1}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{31785BC3-256C-4613-B2F5-A1B0BDDED8C1}.Release|Any CPU.Build.0 = Release|Any CPU
 		{5D3B8245-ADA6-453F-A008-50ED04BFE770}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{5D3B8245-ADA6-453F-A008-50ED04BFE770}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
It's not great these changes were committed... I will put this back in my current branch for means for all devs currently the unit test and benchmarks projects won't build which means people won't be getting build errors for those projects :/

Reply:
Duh! What changes had that affect?

Reply:
It's just these changes in this .sln file, these 2x projects were excluded from the Debug Build (normally done using the 'Configuration Manager' in VS). Was prob done for some reason but the change was overlooked with this merge.